src/structs/lepton_encoder.rs

```diff
@@ -466,42 +471,59 @@ pub fn write_coefficient_block<const ALL_PRESENT: bool, W: Write>(
 
 #[inline(never)] // don't inline so that the profiler can get proper data
 fn encode_edge<W: Write, const ALL_PRESENT: bool>(
-    left: &AlignedBlock,
-    above: &AlignedBlock,
-    here: &AlignedBlock,
+    neighbors_data: &NeighborData,
+    here_tr: &AlignedBlock,
     model_per_color: &mut ModelPerColor,
     bool_writer: &mut VPXBoolWriter<W>,
     qt: &QuantizationTables,
     pt: &ProbabilityTables,
     num_non_zeros_7x7: u8,
     eob_x: u8,
     eob_y: u8,
-) -> Result<()> {
+) -> Result<([i32x8; 8], i32x8, i32x8)> {
+    let q_tr = qt.get_quantization_table_transposed();
+
```

Comment:
this is unsafe since there is an alignment requirement difference. Should be stored as a [u16x8;8] not [u16;64]

Reply:
Will fix.